### PR TITLE
Backup: Release 1.4.1

### PR DESCRIPTION
**Pull request by sdixon194**
#### Changes proposed in this Pull Request:

Releases Jetpack Backup 1.4.1

#### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Jetpack product discussion

n/a
#### Does this pull request change what data or activity we track or use?

no
#### Testing instructions:

* Proof
*

### CODE REVIEW

File: projects/plugins/backup/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 1.4.1 - 2022-09-08
+### Added
+- Include JITMs from My Jetpack [#22452]
```
Comment (kraftbj) [inline projects/plugins/backup/CHANGELOG.md]:
```suggestion
- Include contextual notifications from My Jetpack [#22452]
```

**Review by kraftbj — DISMISSED** [review]
Do we usually use `JITM` in public-facing changelogs?

File: projects/plugins/backup/readme.txt [hunk 2]
```diff
@@ -158,13 +158,13 @@ No, Jetpack Backup does not currently support split site or split home URLs.
...
+### 1.4.1 - 2022-09-08
 #### Added
-- Added Social card to My Jetpack.
+- Include JITMs from My Jetpack
 
 #### Changed
-- Renamed main branch `master` references to `trunk
-- Updated package dependencies.
+- Plugin activation: Only redirect when activating from the Plugins page in the browser
+- Updated package dependencies. [#25713] [#24929] [#24998] [#25048] [#25158] [#25279] [#25315] [#25406]
```
Comment (coder-karen) [inline projects/plugins/backup/readme.txt]: Should these PR links be removed from the readme?

Comment (sdixon194) [reply projects/plugins/backup/readme.txt]: nice catch!

**Review by kraftbj — APPROVED** [review]

**Review by coder-karen — APPROVED** [review]

**Pull request merged**